src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -37,5 +37,10 @@
   <Import Project="..\OrchardCore\OrchardCore.Application.Targets\OrchardCore.Application.Targets.targets" />
 
   <Import Project="..\OrchardCore.Build\OrchardCore.Commons.targets" />
+  
+  <ItemGroup>
+    <PackageReference Include="Lombiq.Tests.UI.AppExtensions" />
+    <PackageReference Include="Lombiq.Tests.UI.Shortcuts" />
+  </ItemGroup>
```

Comment:
Could we not build a separate csproj for tests, i.e. OracheCore.Cms.Web.UITests.csproj or something.... rather than adding in stuff to the main csproj for testing?

Reply:
We could, but it would be a complete copy of this one, and then we'd have to maintain two.